config/bootstrap.php

```diff
@@ -1,17 +1,4 @@
 <?php
-/**
- * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
- * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
- *
- * Licensed under The MIT License
- * For full copyright and license information, please see the LICENSE.txt
- * Redistributions of files must retain the above copyright notice.
- *
- * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
- * @link          http://cakephp.org CakePHP(tm) Project
- * @since         0.10.8
- * @license       http://www.opensource.org/licenses/mit-license.php MIT License
- */
```

Comment:
@markstory what about doing it like this?
https://github.com/FriendsOfCake/crud/blob/master/src/Controller/ControllerTrait.php#L10-L12

At the moment the software created is automatically MIT (or at least parts of it based on cakephp/app and cakephp/bake) and that is most likely not what people want for applications either.